src/test/java/org/broadinstitute/hellbender/tools/exome/AnnotateTargetsIntegrationTest.java

```diff
@@ -40,52 +46,125 @@
     private static final int MEAN_TARGET_SIZE = 50;
     private static final double TARGET_SIZE_STDEV = MEAN_TARGET_SIZE;
     private static final int NUMBER_OF_TARGETS = 100;
+
+    private static final int MIN_BAIT_SIZE = 5;
+    private static final int MAX_BAIT_SIZE = 500;
+    private static final int MEAN_BAIT_SIZE = 20;
+    private static final double BAIT_SIZE_STDEV = MEAN_BAIT_SIZE;
+    private static final int NUMBER_OF_BAITS = 1000; // must be greater than NUMBER_OF_TARGETS
     // End of meta-parameters
 
     private static final Random RANDOM = new Random(1313);
```

Comment:
This is fine here, but I think it is better to set a static random seed and then use this seed to initialize RNGs at the beginning of methods and tests when necessary.  Otherwise, you could imagine that I might add tests in the future that use `RANDOM`, possibly leading to failures when these tests are run in different orders.  I would say that such bugs have accounted for a non-negligible fraction of our Travis failures...so it's good to be preventative!